lib/utils/include/utils/graph/node.h

```diff
@@ -73,9 +73,9 @@ struct GraphView {
     return GraphView(std::make_shared<T>(std::forward<Args>(args)...));
   }
 
-private:
-  GraphView(std::shared_ptr<IGraphView const>);
+  GraphView(std::shared_ptr<IGraphView const> ptr) : ptr(ptr) {}
 
+private:
```

Comment:
```suggestion
```

Reply:
It would be great if you could review your PRs before requesting reviews (such as with `git add -p`) just to avoid small fixes like this making it in to the review (if you're already doing this, no problem--we all miss things sometimes :slightly_smiling_face:)

Reply:
Also why are we making this public? I don't see a reason why this shouldn't be private.